config/config.go

```diff
@@ -45,3 +47,25 @@ func Exists(path string) bool {
 
 	return info.Mode().IsRegular()
 }
+
+func TryLoadRootConfig(dir string) (found bool, cfg hcl.Config, err error) {
```

Comment:
```suggestion
func TryLoadRootConfig(dir string) (cfg hcl.Config, found bool, err error) {
```

Usually on "found" logic the boolean comes after the actual value, like Go maps, but it is not like a rule or pattern, just sharing that it feels odd but probably because I'm not used to it =P. Kinda like actual value first + other complementary information like errors/boolens comes after...but Im probably just stuck with a particular mindset.